src/ducks/categorization/services.js

```diff
@@ -22,6 +22,26 @@ const FAKE_TRANSACTION = {
   label: 'thisisafaketransaction',
   manualCategoryId: '0'
 }
+const TOKENS_TO_REWEIGHT = [
```

Comment:
How those tags work ? How are they extracted ?

Reply:
It comes from here : https://github.com/cozy/cozy-banks/blob/90d9de15b3f8722b3fe2274b09c582df90ec54b4/src/ducks/categorization/services.js#L95
This list is the result of the combinations of (every possible signs tags) x (every possible amount tag)

Remark : it will soon be necessary to simplify them because their redundancy of information and the "amount tag" is sufficient.

Reply:
I just added a comment in code